specification/sections/06-tracking.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Tracking
+description: Specification defining tracking API
+toc_max_heading_level: 4
+---
+
+# 6. Tracking
+
+[![experimental](https://img.shields.io/static/v1?label=Status&message=experimental&color=orange)](https://github.com/open-feature/spec/tree/main/specification#experimental)
+
+## Overview
+
+Experimentation is a primary use case for feature flags.
+In practice, this often means flag variants are assigned to users at random or in accordance with a business rule, while the impact of the assigned variant on some business objective is measured.
+Vendors and custom solutions often support a _tracking_ or _goal measuring_ API to facilitate the measurement of these business objectives.
+
+### Goals
+
+- Develop official terminology to support consistent implementation
+- Specify a flexible API widely compatible with basic vendor functionality
+  - Define tracking event payload
+  - Define tracking event identifier
+  - Support client and server paradigms
+
+### Non-goals
+
+- Creating an experimentation platform
+- Covering every user-tracking use case
```

Comment:
Considering adding as non-goal anything that relies on `rrweb` (https://www.npmjs.com/package/rrweb) such as record and replay functionality

Reply:
I think it's implied

Reply:
I think it's a bit specific to write it down, I think there's a few other similar products we'd also have to note if we noted this one.

Reply:
Yes, I agree that it's implicitly out of scope because `rrweb` is a session replay tool and not an event tracker.